python/pacemaker/_cts/cib.py

```diff
@@ -34,16 +36,6 @@ def __init__(self, cm, factory, tmpfile=None):
 
         self._factory.tmpfile = tmpfile
 
-
-class CIB12(ConfigBase):
-    version = "pacemaker-1.2"
-
-    def __init__(self, cm, factory, tmpfile=None):
-        ConfigBase.__init__(self, cm, factory, tmpfile=tmpfile)
-
-        self._counter = 1
-        self._num_nodes = 0
-
```

Comment:
Yeah, I don't see any point in having multiple classes, not to mention they unnecessarily restrict us to certain schema versions. We can just take the schema name as an argument. If there's ever anything schema-specific we need to do, we can just parse the version number and compare it as needed.